src/main/java/org/myrobotlab/service/interfaces/ServiceLifeCyclePublisher.java

```diff
@@ -0,0 +1,17 @@
+package org.myrobotlab.service.interfaces;
```

Comment:
This should probably be renamed to ServiceLifeCycle not publisher. the publisher interface would be "publishStarted"   not  "started"

Reply:
Sure .. I'll change it to ServiceLifeCycle.  Its a publishing interface, however registered and released have been around for bajillion years (as string values), so even though it is a publishing interface I'll wait to refactor to publishRegistered etc for another refactor)